pkg/controller/command/sdscomm/sdscomm.go

```diff
@@ -6,20 +6,18 @@ SPDX-License-Identifier: Apache-2.0
 package sdscomm
 
 import (
-	"crypto/rand"
 	"encoding/json"
 	"fmt"
 	"strings"
 
-	"github.com/btcsuite/btcutil/base58"
+	"github.com/trustbloc/edv/pkg/edvutils"
```

Comment:
note: this package would be better named in the singular rather than the plural.

Reply:
Created trustbloc/edv#100.